package.json

```diff
@@ -8,12 +8,10 @@
   "version": "1.0.0",
   "description": "Skyscanner full-stack recruitment test.",
   "scripts": {
-    "install": "(cd ./client && npm install) && (cd ./server && npm install)",
-    "client": "(cd ./client && npm start)",
+    "install": "(cd ./client && npm install)",
```

Comment:
If we have removed the server aspect - we should get rid of the `client/` folder. This top level `package.json` is redundant now